{{cookiecutter.repostory_name}}/envs/dev/.env.template

```diff
@@ -38,3 +38,7 @@ CSP_MANIFEST_SRC="{{cookiecutter.csp_manifest_src}}"
 CSP_WORKER_SRC="{{cookiecutter.csp_worker_src}}"
 CSP_BLOCK_ALL_MIXED_CONTENT={{cookiecutter.csp_block_all_mixed_content}}
 CSP_EXCLUDE_URL_PREFIXES={{cookiecutter.csp_exclude_url_prefixes}}
+BACKUP_B2_BUCKET=
+BACKUP_B2_KEY_ID=
+BACKUP_B2_KEY_SECRET=
+BACKUP_ROTATE_KEEP_LAST=
```

Comment:
the variable name should clearly indicate that those are local snapshots that we are going to keep - not remote. There is a b2cli option named in a similar way.

Reply:
Renamed